rancher-config/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '2'
+services:
+  shis-etl:
+    image: nhsuk/sexual-health-information-etl:${DOCKER_IMAGE_TAG}
+    environment:
+      ETL_NAME: 'shis-data-etl'
+      INITIAL_LAST_RUN_DATE: '${INITIAL_LAST_RUN_DATE}'
```

Comment:
Would one ETL want to have a different last run date than the other?

Reply:
Potentially, I was under the impression that everything in these buckets were uploaded from the same source sheet, but I'm not certain so will expose them as individual variables.